fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2Reorg.java

```diff
@@ -16,6 +16,10 @@
 
 /**
  * Reorg the schema.table
+ *
+ * Be sure to complete all database operations and release all locks before you invoke REORG.
+ * @see <a href="https://www.ibm.com/support/knowledgecenter/SSEPGG_11.5.0/com.ibm.db2.luw.admin.cmd.doc/doc/r0001966.html">
+ *      https://www.ibm.com/support/knowledgecenter/SSEPGG_11.5.0/com.ibm.db2.luw.admin.cmd.doc/doc/r0001966.html</a>
```

Comment:
this does remember me of one question, should we force all db connections to drop first before we run reorg during migration?

Reply:
this is an excellent question.  the migration seems to work now without that.  my biggest concern is that its actually committing the migrations under the covers because that would break our "do all the migrations within a single transaction" model.
still, i'd prefer to punt on that one for now so we can get this in for 4.1.0...maybe open it as a separate issue?

Reply:
I think we should probably add to the documentation stating that.

Reply:
which documentation.  i did add it here in the javadoc but not in the readme

Reply:
I was thinking to the design considerations in Db2MultitenantSchema... Essentially what we discussed is 'offline' updates.

Reply:
I updated the doc